library/src/helpers/parser.ts

```diff
@@ -68,10 +70,7 @@ class Parser {
       return { data: null, error: { message: err.message } };
     }
 
-    if (
-      err.parsedJSON &&
-      !this.isCorrectSchemaVersion(err.parsedJSON.asyncapi)
-    ) {
+    if (err.parsedJSON && err.parsedJSON.asyncapi.startsWith('1')) {
```

Comment:
Please revert this to `isCorrectSchemaVersion` function.

Reply:
- `isCorrectSchemaVersion` only checks the first character e.g. `-1.2.3` is not a correct schema version but the function would return `true`
- it is oneliner used just in this place